js/components/conceptset/ConceptSetStore.js

```diff
@@ -175,6 +178,7 @@ define([
 			const identifiers = concepts.map(c => c.CONCEPT_ID);
 			try {
 				const data = await vocabularyService.getMappedConceptsById(identifiers);
+				await vocabularyService.loadDensity(data);
```

Comment:
The source codes does not show record counts (because source codes aren't actual records, but show the relationship between a mapped source code).   Does this PR change the 'Included soruce codes' view to display row counts?